### PR TITLE
feat: add customer examples for invoice pdf and medical report image

### PR DESCRIPTION
**Pull request by jojortz**
**Feature**
Add new demos:
- `pdf_to_key_value.ipynb` and `async_pdf_to_key_value.ipynb`: Demo extracting key-value from example Invoice document
- `image_to_key_value.ipynb` and `async_image_to_key_value.ipynb`: Demo extracting key-value pairs from example medical report image

**Testing**
- ran automated tests in `test/test.py` via pre-commit
- ran new notebooks

### CODE REVIEW

**Comment by CambioML** [comment]
Please fix broken test cases.

**Comment by CambioML** [comment]
Please fix broken test cases.

File: examples/sample_data/test_medical_report.jpeg
Comment (CambioML) [inline examples/sample_data/test_medical_report.jpeg:1]: You should consider to put sensitive information even this in a S3 bucket and put a .env to ask people to put access key and secret key to download.

Comment (CambioML) [reply examples/sample_data/test_medical_report.jpeg:1]: Is this issue addressed?

**Comment by CambioML** [comment]
> Please fix broken test cases.

What is the status of the broken unittest?

**Comment by jojortz** [comment]
All test cases passed.

This is already updated to be `extract_key_value` to match cambio-core.

**Review by lingjiekong — APPROVED** [review]
LGTM.

**Pull request merged**